### PR TITLE
Added NUL option to Backup docs

### PR DESCRIPTION
**Pull request by way0utwest**
Added the NUL device as an option with a short description that input is discarded.

### CODE REVIEW

**Comment by PRMerger3** [comment]
@way0utwest : Thanks for your contribution to the SQL documentation! The author, @JennieHubbard, has been notified to review your proposed change.

**Comment by craigg-msft** [comment]
Thank you @way0utwest for the help. @edmacauley could you please review.

**Comment by edmacauley** [comment]
Hi I’m not the regular SME on this subject, but I believe that the NUL option is meant to be undocumented.  Do you have any reference documentation we can point to for more information?

**Comment by way0utwest** [comment]
That's an interesting question. I have no knowledge of whether NUL is supposed to be undocumented, but it's a standard DOS/CLI output that's available. This was noted in earlier BOL versions as an option. I don't know if it's been deprecated, but it's possible I've missed this.

I'm not sure why this would be undocumented, as it's a valid file to the file system on the OS. It is useful in testing backups, initializing a backup for something like AGs, etc.

**Comment by johlju** [comment]
If we should document this then maybe it should state that NUL should only to be used for testing, and not recommended in production environments where one want a backup?

**Comment by way0utwest** [comment]
I can make that change and resubmit a PR if that's appropriate.

**Comment by johlju** [comment]
You could wait and see what @JennieHubbard has to say about it. 

Btw, you don't have to resubmit the PR. Just push commits to the same branch in your fork and this PR will update.

**Comment by JennieHubbard** [comment]
@way0utwest @edmacauley @johlju Either option should be fine. Either make the changes as a commit on this pr, or close this one and make your changes in a new pr. @way0utwest do whichever you find easiest to do, and thanks again for your contributions! 👍

**Comment by way0utwest** [comment]
Thanks, Jennifer.

I updated my branch with a note that includes the verbiage that this is for testing only.

File: docs/t-sql/statements/backup-transact-sql.md [hunk 1]
```diff
@@ -226,15 +226,19 @@ TO \<backup_device> [ **,**...*n* ]
...
+ { DISK | TAPE | URL} **=** { **'***physical_device_name***'** | **@***physical_device_name_var* | NUL }  
+ Specifies a disk file or tape device, or a Windows Azure Blob storage service. The URL format is used for creating backups to the Windows Azure storage service. For more information and examples, see [SQL Server Backup and Restore with Microsoft Azure Blob Storage Service](../../relational-databases/backup-restore/sql-server-backup-and-restore-with-microsoft-azure-blob-storage-service.md). For a tutorial, see [Tutorial: SQL Server Backup and Restore to Windows Azure Blob Storage Service](~/relational-databases/tutorial-sql-server-backup-and-restore-to-azure-blob-storage-service.md). 
+
+Note: The NUL disk device will discard all information sent to it and should only be used for testing. This is not for production use.
```
Comment (johlju) [inline docs/t-sql/statements/backup-transact-sql.md]: Maybe we should make this in the same format as the note on line 195-196?

**Comment by way0utwest** [comment]
Added a note there. Not sure it's needed, but it does make the doc more complete.

**Comment by craigg-msft** [comment]
Thank you all - @way0utwest @johlju  @JennieHubbard  @edmacauley  merging.

**Pull request merged**